### PR TITLE
Deny webcrawlers to index ab splittest pages

### PR DESCRIPTION
**Pull request by TPRobots**
Pull request opened by github-pullrequestcreator.

### CODE REVIEW

File: src/splittest.ts [hunk 1]
```diff
@@ -61,7 +61,8 @@ export class SplitTest {
             (typeof config.globalCondition !== 'function' ||
                 config.globalCondition(userAgentInfo)) &&
             (typeof this.condition !== 'function' ||
-                this.condition(userAgentInfo))
+                this.condition(userAgentInfo)) &&
+            !/bot|googlebot|crawler|spider|robot|crawling/i.test(navigator.userAgent)
```
Comment (b-dur) [inline src/splittest.ts]: the **bot** statement will catch google**bot** and ro**bot**.

File: src/splittest.ts [hunk 2]
```diff
@@ -61,8 +61,7 @@ export class SplitTest {
             (typeof config.globalCondition !== 'function' ||
                 config.globalCondition(userAgentInfo)) &&
             (typeof this.condition !== 'function' ||
-                this.condition(userAgentInfo)) &&
-            !/bot|googlebot|crawler|spider|robot|crawling/i.test(navigator.userAgent)
```
Comment (b-dur) [inline src/splittest.ts]: thanks for letting me code review previous commit 😒

File: src/useragentinfo.ts [hunk 3]
```diff
@@ -37,15 +37,22 @@ function isMobile() {
...
 }
 
+function isBot() {
+    const ua = navigator.userAgent || navigator.vendor;
+    return /bot|googlebot|crawler|spider|robot|crawling/i.test(ua)
```
Comment (b-dur) [inline src/useragentinfo.ts]: same as my first comment

**Comment by Tudes** [comment]
@b-dur I had a chat with @thomicdk and this PR will be closed

**Pull request closed without merging**